### DISCUSSION TITLE
Question aboout DC2_exp.ipynb

### DISCUSSION THREAD

**Discussion by YicunDuanUMich** [prob-ml/bliss#982] [Q&A]
@XinyueLi1012 
hi xinyue,

could you please give some instructions on how to reproduce the results in DC2_exp.ipynb? It seems that this file can't be run directly.

yicun

**Comment by XinyueLi1012** [comment]
Hi Yicun,

When we train with dc2 data, due to memory limit, we split the original image into smaller ones (usually one 4k*4k image to 250 80*80 images), but  for plotting and comparing some prediction results, I used the full-sized image. To do this, I temporarily modify the `DC2` class in `dc2.py` to bypass the splitting (from line 113 to 146) and return the complete image along with the full catalog (and also wcs, matchid).

Here are some other hints about dc2:

1. Make sure that your have set the right `set_root_dir` to load the catalog using `GCRCatalogs` package
2. Sometimes `to_tile_catalog` function in `catalog.py` won't work for dc2 data (if you want to re-train the model)

**Reply by YicunDuanUMich** [reply]
Thank you for your reply. I have one more question. Where can we find the `matchid`? We can't find it in the current dataset. Besides, i will appreciate it if you can provide the path to your training and testing datasets.

**Reply by XinyueLi1012** [reply]
The `matchid` could be extracted from the dc2 catalog file (which could be generated using `DC2_galaxy_psf_params.ipynb`). The dataset I use could be found in `/nfs/turbo/lsa-regier/lsstdesc-public/dc2...`

**Reply by YicunDuanUMich** [reply]
Thanks!
